City-of-Helsinki/helsinki-paatokset
# Pp 133 meetings calendar

Tickets: https://helsinkisolutionoffice.atlassian.net/browse/PP-133, https://helsinkisolutionoffice.atlassian.net/browse/PP-250

**How to test** 

Checkout branch and run `drush cim -y; drush cr`

Create a landing page and name it Kokouskalenteri, the new block will automatically render to a page that has /kokouskalenteri as url path (can be changed if needed). Add a hero area to the landing page if you want it to have the gold header area.

Test the calendar, check the [styles](https://app.abstract.com/projects/9e324ba2-c4b2-45b5-93d9-92f00c1c688e/branches/master/commits/latest/files/7bad9819-85bc-4fe1-b2bb-e53b8b35d442/layers/7F796650-1D89-4624-9E00-D8A343CD2F04?collectionId=94278a45-ac4b-49c8-910a-24848370526e&collectionLayerId=3dd6d83f-2c28-407d-8891-dfb25d7b2a0d&mode=build&selected=root-C314ADFD-683F-4755-BF45-269AB69F6EB4) and check the code.

Keep in mind that one on the breakpoints is not working properly so the styles break below 1248px for a bit.

Is the block too slow when loading the calendar? Should it be faster?
